query/src/datasources/database/github/util.rs

```diff
@@ -0,0 +1,59 @@
+//  Copyright 2021 Datafuse Labs.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+use common_exception::ErrorCode;
+use common_exception::Result;
+use common_meta_types::TableInfo;
+use octocrab::Octocrab;
+use reqwest::header::AUTHORIZATION;
+
+use crate::datasources::database::github::database::OWNER;
+use crate::datasources::database::github::database::REPO;
+
+pub fn create_github_client() -> Result<Octocrab> {
```

Comment:
Why the comments table need the github token and others not?

```
mysql> select * from databend_comments;
ERROR 1105 (HY000): Code: 7002, displayText = Github Access Token is not set, see how to set at https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token, environment variable not found.

```

Reply:
all of them will be needed because github api has a very slow rate limit on unauthorized user. see here https://github.com/datafuselabs/databend/issues/2975#issuecomment-978888940

Reply:
all reading from github will call the method `util::create_github_client()`, they will all check whether `GITHUB_TOKEN` is set